DatamartManagementService/DatamartManagementService.API/Program.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DatamartManagementService.API
+{
+    public class Program
+    {
+        public static void Main(string[] args)

```

Comment:
I don't think we have to add API layer for this.
This will just be a solution with jobs running to pull data from one database to another.